tests/downloads/test_mixed_content_download_via_https.py

```diff
@@ -13,11 +13,14 @@
 def test_case():
     return "1756722"
 
+@pytest.fixture()
+def delete_files_regex_string():
+    return r"download"
```

Comment:
Can we write a more specific regex? Don't want to accidentally blast a bunch of unrelated downloads.

Reply:
wouldwould '\bdownload\b' be good?

Reply:
It would be better, I think